src/api/public/apidocs-new/paths/build_project_name_repository_name_architecture_name_package_name_buildinfo.yaml

```diff
@@ -32,3 +32,96 @@ get:
 
   tags:
     - Build
+
+post:
+  summary: Return build information about a build description would produce.
+  description: |
+    Get build information using the file passed in the request body.
+
+    Despite using the method `POST`, this endpoint doesn't alter any data.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/repository_name.yaml'
+    - $ref: '../components/parameters/architecture_name.yaml'
+    - in: path
+      name: package_name
+      schema:
+        type: string
+      required: true
+      description: |
+        Package name.
```

Comment:
Should we mention something about _multibuild (`osc -M foo`) that is probably appended as `:foo` here?

e.g. in openSUSE:Factory/avahi, a `osc -v build --noservice -M qt5 standard` calls
makeurl: https://api.opensuse.org ['build', 'openSUSE:Factory', 'standard', 'x86_64', 'avahi:qt5', '_buildinfo'] ['add=gdb', 'add=vim', 'add=strace', 'add=less']

That `add=` param would be useful to document, too.

Reply:
Just to make sure we are talking about the same API endpoint here (`POST /build/{project_name}/{repository_name}/{architecture_name}/{package_name}/_buildinfo`), could you run that `osc` command with the `-H` flag, to see what API calls are involved?

I fear that we are talking about another endpoint. It also could belong to the already documented [GET one](https://api.opensuse.org/apidocs-new/index#/Build/get_build__project_name___repository_name___architecture_name___package_name___buildinfo).

Reply:
```
Getting buildinfo from server and store to /space/osc/openSUSE:Factory/avahi/.os
c/_buildinfo-standard-x86_64.xml
makeurl: https://api.opensuse.org ['build', 'openSUSE:Factory', 'standard', 'x86
_64', 'avahi:qt5', '_buildinfo'] ['add=gdb', 'add=vim', 'add=strace', 'add=less'
]


-- POST https://api.opensuse.org/build/openSUSE:Factory/standard/x86_64/avahi:qt
5/_buildinfo?add=gdb&add=vim&add=strace&add=less
send: b'POST /build/openSUSE:Factory/standard/x86_64/avahi:qt5/_buildinfo?add=gd
b&add=vim&add=strace&add=less HTTP/1.1\r\nAccept-Encoding: identity\r\nAccept: a
pplication/xml\r\nContent-type: application/octet-stream\r\nContent-length: 3319
8\r\nHost: api.opensuse.org\r\nUser-agent: osc/0.182.0\r\nConnection: close\r\n\
r\n'
send: b'#\n# spec file\n#\n# Copyright (c) 2023 SUSE LLC\n#\n# All modifications
 and additions to the file contributed by third parties\n# remain the property...
```